docs/guides/VTEX-Platform-Overview/developer-experience.md

```diff
@@ -0,0 +1,136 @@
+
+- [**Releasing a new app version**](https://developers.vtex.com/docs/guides/vtex-io-documentation-releasing-a-new-app-version): Marks the beginning by releasing the app using Git, updating the version in `manifest.json`, and documenting changes in CHANGELOG.md.
+- [**Publishing**](https://developers.vtex.com/docs/guides/vtex-io-documentation-publishing-an-app): The app version becomes a candidate version, ready for testing and validation in a production workspace.
+- [**Testing**](https://developers.vtex.com/docs/guides/ab-tests): Run [A/B tests](#ab-tests) and others, such as performance and security testing.
+- [**Deploying**](https://developers.vtex.com/docs/guides/vtex-io-documentation-deploying-the-app-stable-version): Update the app version across all accounts where the app is installed.
+- [**Promotiong**](https://developers.vtex.com/docs/guides/vtex-io-documentation-promoting-a-workspace-to-master): Optional step to promote the production workspace to master.
+
+>ℹ️ To learn about app versioning in VTEX IO, refer to [SemVer standard](https://semver.org/) and our [Understanding app versioning](https://developers.vtex.com/docs/guides/vtex-io-documentation-releasing-a-new-app-version) article.
+
+Our deployment process integrates an automated Continuous Integration and Continuous Delivery (CI/CD) framework, facilitating seamless building and testing whenever changes occur in the source code.
+
+Through this CI/CD pipeline, we ensure swift and dependable delivery of new features and updates, while maintaining high standards of code quality.
+
+## Storefront development
+
+VTEX provides different options for storefront development, each with its own characteristics, according to your business needs:
+
+- [**FastStore**](https://developers.vtex.com/docs/guides/faststore/docs-what-is-faststore): innovative storefront technology based on the [Jamstack](https://jamstack.org/) architecture. This technology prioritizes performance and stability, offering development teams an easily maintainable solution for editing store pages.
+- [**Store Framework**](https://developers.vtex.com/docs/guides/storefront-implementation): emphasizes a low-code, composable commerce model, allowing integration of [VTEX IO apps](https://developers.vtex.com/docs/vtex-io-apps) for dynamic storefronts.
+- [**Legacy CMS Portal**](https://developers.vtex.com/docs/guides/vtex-io-documentation-migrating-storefront-from-legacy-to-io): VTEX’s inaugural storefront technology, that relies on HTML, CSS, and JavaScript, with code management exclusively through the VTEX Admin. This technology is no longer available to new stores starting with VTEX.
+
+### Learn more
+
+- [Getting started with storefront solutions](https://developers.vtex.com/docs/guides/getting-started-with-storefront-solutions)
+- [FastStore onboarding](https://developers.vtex.com/docs/guides/faststore-onboarding)
+- [Store Framework onboarding](https://developers.vtex.com/docs/guides/store-framework-onboarding)
```

Comment:
Since all these guides are from Developer Portal, they should be added as seeAlso in the frontmatter